inyoka_theme_default/static/style/inyoka/planet.less

```diff
@@ -10,16 +10,10 @@
 
 .planet {
   &-newestarticles {
-    .list-inline {
-      text-align:justifed;
-
-      li {
-        padding: 0;
-
-        &:not(:last-child):after {
-          color: @gray-light;
-          content: ' |';
```

Comment:
To only use bootstrap's `.list-inline`. Thus, dropping the old style from uu.de which is not very typographic btw.